SQL/2016-03-10-MoveSelectedTo_files_qcstatus_table.sql

```diff
@@ -0,0 +1,10 @@
+/* ALTER files_qcstatus table to add a Selected column */
```

Comment:
These should be -- at the start of the line for a comment, not /* ... */. /* is a php comment, -- is an SQL comment.